src/input.rs

```diff
@@ -113,7 +113,7 @@ pub fn select<T: Display>(
         }
     } else {
         Select::new(desc, options)
-            .with_page_size(page_size())
+            .with_page_size(page_size() / 2) //Fixing bug with page size
```

Comment:
Just bizarre. I pulled this branch down and ran it locally. Looks like it's working well :+1:

Reply:
Not sure if we're rendering the menu twice somewhere or rendering two different menus (or the prompt and hte selection differently), but this works well so we'll go with it for now - thx!